package.json

```diff
@@ -37,7 +43,15 @@
     "@types/node": "^16.11.13",
     "@types/w3c-web-hid": "^1.0.2",
     "@types/w3c-web-usb": "^1.0.5",
+    "@typescript-eslint/eslint-plugin": "^5.10.1",
+    "@typescript-eslint/parser": "^5.10.1",
+    "eslint": "^8.7.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "install": "^0.13.0",
     "jest": "^27.4.7",
+    "npm": "^8.4.0",
```

Comment:
We can remove this dependency as this supplied with Node.js itself and considered a prerequisite for this project.